test/torchaudio_unittest/common_utils/rnnt_utils.py

```diff
@@ -189,13 +189,17 @@ def compute_with_numpy_transducer(data):
 
 
 def compute_with_pytorch_transducer(data):
+    fused_log_softmax = data["fused_log_softmax"] if "fused_log_softmax" in data else True
```

Comment:
If `data` is `dict`.

```suggestion
    fused_log_softmax = data.get("fused_log_softmax", True)
```